test/e2e-common.sh

```diff
@@ -92,7 +92,7 @@ function knative_setup() {
 
   install_feature_cm || fail_test "Could not install features configmap"
 
-  create_knsubscribe_rolebinding || fail_test "Could not create knsubscribe rolebinding"
+  #create_knsubscribe_rolebinding || fail_test "Could not create knsubscribe rolebinding"
```

Comment:
```suggestion
  create_knsubscribe_rolebinding || fail_test "Could not create knsubscribe rolebinding"
```
I guess a leftover?
Otherwise LGTM

Reply:
arg - damn - my tmp work-around :smile:

Reply:
:laughing: